entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+
+# Create the user with ssh acces
+useradd -d "/home/${SSH_USER}" -m -s "/bin/bash" "${SSH_USER}"
+mkdir -p /home/${SSH_USER}/.ssh/
+
+# Configure ssh_config
+mkdir -p /var/run/sshd
+echo "    Port 22" >> /etc/ssh/ssh_config
```

Comment:
@ruiztulio 
Do you tell me one example of the command `sed` ?
Because actually the original configuration file `/etc/ssh/ssh_config` not has that `lines` 

![image](https://user-images.githubusercontent.com/1387970/28333678-a9de7c12-6bc6-11e7-80d4-248e1ab6ecc9.png)